kafka-ui-react-app/src/lib/yupExtended.ts

```diff
@@ -105,8 +105,16 @@ export const topicFormValidationSchema = yup.object().shape({
     .max(2147483647)
     .required()
     .typeError('Number of partitions is required and must be a number'),
-  replicationFactor: yup.string(),
-  minInSyncReplicas: yup.string(),
+  replicationFactor: yup
+    .number()
+    .transform((_, val) => (isNaN(val) ? 0 : Number(val)))
```

Comment:
please check this error.

Unexpected use of 'isNaN'. Use Number.isNaN instead https://github.com/airbnb/javascript#standard-library--isnan

Reply:
No longer a problem.